src/pkg/scan/sbom/sbom.go

```diff
@@ -96,22 +96,45 @@ func (v *scanHandler) PostScan(ctx job.Context, sr *v1.ScanRequest, _ *scanModel
 		Artifact: sr.Artifact,
 	}
 	// the registry server url is core by default, need to replace it with real registry server url
-	scanReq.Registry.URL, scanReq.Registry.Insecure = v.RegistryServer(ctx.SystemContext())
+	scanReq.Registry.URL, scanReq.Registry.Insecure = h.RegistryServer(ctx.SystemContext())
 	if len(scanReq.Registry.URL) == 0 {
 		return "", fmt.Errorf("empty registry server")
 	}
 	myLogger := ctx.GetLogger()
 	myLogger.Debugf("Pushing accessory artifact to %s/%s", scanReq.Registry.URL, scanReq.Artifact.Repository)
-	dgst, err := v.GenAccessoryFunc(scanReq, sbomContent, v.annotations(), sbomMimeType, robot)
+	dgst, err := h.GenAccessoryFunc(scanReq, sbomContent, h.annotations(), sbomMimeType, robot)
 	if err != nil {
 		myLogger.Errorf("error when create accessory from image %v", err)
 		return "", err
 	}
-	return v.generateReport(startTime, sr.Artifact.Repository, dgst, "Success", s)
+	return h.generateReport(startTime, sr.Artifact.Repository, dgst, "Success", s)
+}
+
+// URLParameter defines the parameters for scan report url
+func (h *scanHandler) URLParameter(_ *v1.ScanRequest) (string, error) {
+	return fmt.Sprintf("sbom_media_type=%s", url.QueryEscape(sbomMediaTypeSpdx)), nil
+}
+
+// RequiredPermissions defines the permission used by the scan robot account
+func (h *scanHandler) RequiredPermissions() []*types.Policy {
```

Comment:
Fore better readability, maybe move `func (h *scanHandler) RequiredPermissions() []*types.Policy { }` to be after `RequestProducesMineTypes` and before `RequestParameters`, instead of having it after `func (h *scanHandler) URLParameter(_ *v1.ScanRequest) (string, error) {}`